oneflow/ir/oneflow-translate/lib/OneFlow/Importer.cpp

```diff
@@ -380,6 +397,39 @@ LogicalResult ParseNdSbpFromAttr(::llvm::ArrayRef<Attribute> nd_sbp_attr,
   return success();
 }
 
+Attribute ConvertNdSbpToPsig(Builder& builder,
+                             const ::google::protobuf::RepeatedPtrField<std::string>& nd_sbp,
+                             int nd_size) {
+  auto ctx = builder.getContext();
+  std::vector<mlir::Attribute> outputs_vec;
+  for (const auto& sbp_data : nd_sbp) {
+    ::oneflow::SbpParallel sbp;
+    ParseSbpParallelFromString(sbp_data, &sbp);
+    Attribute attr;
+    if (sbp.has_split_parallel()) {
+      attr = sbp::SplitAttr::get(ctx, sbp.split_parallel().axis());
+    } else if (sbp.has_broadcast_parallel()) {
+      attr = sbp::BroadcastAttr::get(ctx);
+    } else if (sbp.has_partial_sum_parallel()) {
+      attr = sbp::PartialSumAttr::get(ctx);
+    } else {
+      llvm::errs() << "unsupported sbp: nd_sbp from string";
```

Comment:
```suggestion
      llvm::errs() << "Unsupported sbp type from nd_sbp";
```

这句话可以适当refine一下，感觉语序有点奇怪。